lib/govspeak.rb

```diff
@@ -5,6 +5,7 @@
 require 'kramdown'
 require 'kramdown/parser/kramdown_with_automatic_external_links'
 require 'rinku'
+require "govuk_publishing_components"
```

Comment:
Tiny annoying nitpick - this is in double quotes but the other requires have single quotes.

Reply:
Argh! Thanks for catching, I've fixed another one of those in the test files.